google/cloud/spanner_v1/snapshot.py

```diff
@@ -538,7 +538,7 @@ def _get_streamed_result_set(
         iterator = _restart_on_unavailable(
             restart,
             request,
-            "CloudSpanner.ReadWriteTransaction",
+            "CloudSpanner.execute_sql",
```

Comment:
```suggestion
            "CloudSpanner.execute_streaming_sql",
```
since it is a streaming call